app/views/repositories/_form.html.erb

```diff
@@ -0,0 +1,88 @@
+<div class="row repo-show white" id="repository_update">
+  <div class="container z-depth-1">
+    <div class="row">
+      <div class="col l12">
+        <h4 class="para-text" style="margin-left: 10px">
+          <%= @repository.name.titleize %></h4>
+      </div>
+      <div class="col s12 poc_form">
+        <div class="row">
+          <div class="col l5">
+            <div style="width:300">
+              <div class="hovereffect" onclick="$('#poc_image').trigger('click');">
+                <%= image_tag @repository.poc_image, width: 300, id: "image" %>
+                <div class="overlay">
+                  <h2>Image</h2>
+                  <p>
+                    <a href="#">EDIT</a>
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col l7">
+            <div class="input-field col l12">
+              <textarea id="description" class="materialize-textarea poc_desc_text_area"><%= @repository.description %></textarea>
+              <label for="textarea1">Description</label>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col l12">
+            <label for="chips">Tags</label>
+            <div id="div_for_tags">
+              <% @repository.tag_list.each do |tag| %>
+              <div class="chip"><%= tag %></div>
+              <% end %>
+            </div>
+            <input type="text" class="chips chips-placeholder" id="tag_list" value="<%= @repository.tag_list %>">
+          </div>
+        </div>
+        <div class="row">
+          <div class="col l12">
+            <button class="btn waves-effect waves-light right" style="float:left" name="action" onclick="displayPreview();">
+              PREVIEW<i class="material-icons right">send</i>
+            </button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div id="repository_preview" hidden=true>
+  <%= render 'repository_preview' %>
+</div>
+
+<script>
+  var tags_string = ""
+
+  $("#tag_list").keyup(function (e) {
```

Comment:
shouldn't it be bound after document ready?

And write JS in object literal pattern.